client/src/App.js

```diff
@@ -1,5 +1,11 @@
+import Home from './Pages/Home';
+
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
```

Comment:
I think you can remove the className here as you do not import any stylesheet :)

Reply:
Done :)